bucket/registry-explorer.json

```
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://ericzimmerman.github.io",
+    "description": "Windows registry viewer.",
+    "version": "1.5.0.0",
+    "license": "MIT",
+    "url": "https://f001.backblazeb2.com/file/EricZimmermanTools/RegistryExplorer_RECmd.zip",
+    "hash": "c4405304a5327fb9fd7c5a04582ab2f23d9751b858fb82d6e81f26b828cb4e97",
+    "extract_dir": "RegistryExplorer",
+    "bin": "RECmd.exe",
```

```suggestion
    "bin": [
        "RECmd.exe",
        "RegistryExplorer.exe"
    ],
```
In general, let’s give users the option to run GUI apps directly from the command-line, ok?